test/e2e/benchmark/benchmark.go

```diff
@@ -100,26 +98,27 @@ func (b *BenchmarkTest) SetupNodes() error {
 		testnet.WithMempoolMaxTxsBytes(1*testnet.GiB),
 		testnet.WithMempoolMaxTxBytes(8*testnet.MiB),
 	))
-	if b.manifest.PushTrace {
-		log.Println("reading trace push config")
-		if pushConfig, err := trace.GetPushConfigFromEnv(); err == nil {
-			log.Print("Setting up trace push config")
-			envVars := map[string]string{
-				trace.PushBucketName: pushConfig.BucketName,
-				trace.PushRegion:     pushConfig.Region,
-				trace.PushAccessKey:  pushConfig.AccessKey,
-				trace.PushKey:        pushConfig.SecretKey,
-				trace.PushDelay:      fmt.Sprintf("%d", pushConfig.PushDelay),
-			}
-			for _, node := range b.Nodes() {
-				for key, value := range envVars {
-					if err = node.Instance.Build().SetEnvironmentVariable(key, value); err != nil {
-						return fmt.Errorf("failed to set %s: %v", key, err)
-					}
-				}
-			}
-		}
-	}
+	// TODO: waiting on "github.com/cometbft/cometbft/pkg/trace" to be implemented
+	// if b.manifest.PushTrace {
+	// 	log.Println("reading trace push config")
+	// 	if pushConfig, err := trace.GetPushConfigFromEnv(); err == nil {
+	// 		log.Print("Setting up trace push config")
+	// 		envVars := map[string]string{
+	// 			trace.PushBucketName: pushConfig.BucketName,
+	// 			trace.PushRegion:     pushConfig.Region,
+	// 			trace.PushAccessKey:  pushConfig.AccessKey,
+	// 			trace.PushKey:        pushConfig.SecretKey,
+	// 			trace.PushDelay:      fmt.Sprintf("%d", pushConfig.PushDelay),
+	// 		}
+	// 		for _, node := range b.Nodes() {
+	// 			for key, value := range envVars {
+	// 				if err = node.Instance.Build().SetEnvironmentVariable(key, value); err != nil {
+	// 					return fmt.Errorf("failed to set %s: %v", key, err)
+	// 				}
+	// 			}
+	// 		}
```

Comment:
I think we can remove this as we don't use traces in knuu tests atm really. the files should be uploaded by knuu automatically to some s3 bucket if it does need to occur